validators/register-validator-node.md

```diff
@@ -23,23 +23,15 @@ To be able to set up an SGX Wallet, validators are required to have an SGX compa
 SKALE will have two types of SGX operations:
 
 -   **Local (Secure)**: Wallet running on the same server as sub-node.  
--   **Network**: Sub-node talks to SGX wallet over the SKALE Network. The validator is responsible for securing the connection. If validator is planning to have a separate SGX compatible node than the Blockchain node, SGX Wallet node doesn't have to have the same hardware requirements as the sub-node. SGXWallet doesn't require a lot of computational power. After setting up the Network SGX node, enable SSL sertification before adding the url to configuration in SKALE Node Set up.
+-   **Network**: Sub-node talks to SGX wallet over the SKALE Network. The validator is responsible for securing the connection. If validator is planning to have a separate SGX compatible node than the Blockchain node, SGX Wallet node doesn't have to have the same hardware requirements as the sub-node. SGXWallet doesn't require a lot of computational power. After setting up the Network SGX node, enable SSL certification before adding the url to configuration in SKALE Node Set up.
```

Comment:
Two things:
- is Blockchain Node == SKALE Node?
- should SGXWallet == SGX Wallet?

Reply:
I have another push coming today and will be fixed with that